src/cli.rs

```diff
@@ -61,7 +64,8 @@ pub struct Opts {
     ignore: (),
 
     ///Show search results from files and directories that would otherwise be
-    /// ignored by '.gitignore' files. The flag can be overridden with --ignore-vcs.
+    /// ignored by '.gitignore' files or the rule to exclude .git/.
+    /// The flag can be overridden with --ignore-vcs.
```

Comment:
My suggestion:
```
    /// Show search results from '.git/' folders and results that would
    /// otherwise be ignored by '.gitignore' rules.
```